homework-submission/week4hw/src/App.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import Form from "./components/Form";
+import TodoItem from "./components/TodoItem";
+import "./App.css";
+import { observer, inject } from "mobx-react";
+
+@inject("todoListStore")
+@observer
+class App extends Component {
+  render() {
+    const { listItems } = this.props.todoListStore;
+    let renderList;
+    if (listItems.length > 0) {
+      renderList = listItems.map((todo, index) => {
+        return <TodoItem key={index} index={index} todo={todo} />;
+      });
+    } else {
```

Comment:
That's true

Reply:
I had to reread the code again to be sure, i noticed it's not useless here, since the return is inside the `map` it has no affect over the function, it doesn't break the function so without `else`, `renderList` is going to be always equal to `<p> No items... </p>`

Reply:
Aha, Sorry my bad 🙃